.github/workflows/ruby-cache.yml

```diff
@@ -1,17 +0,0 @@
-name: Ruby Cache
-
-on:
-  push:
-    branches: [master]
-
-jobs:
-  cache:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check out files
-        uses: actions/checkout@v2
-
-      - name: Set up Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          bundler-cache: true
```

Comment:
No longer needed, since we cache in the `test` job that now runs against merge to master.